### PR TITLE
Add information about multi-user systems

### PR DESCRIPTION
**Pull request by KrauseFx**
Taken from https://github.com/fastlane/fastlane/pull/5902, thanks @PaulTaykalo

### CODE REVIEW

File: docs/intro/Appfile.md [hunk 1]
```diff
@@ -63,3 +63,49 @@ If you want to access those values from within your `Fastfile` use
...
+if File.exist?(filename)
+    puts "Using local Appfile #{filename}"
+    eval(File.read(filename)) 
+end
+```
+
+.gitignore:
+
+```sh
+apple_id.txt
```
Comment (milch) [inline docs/intro/Appfile.md]: This should say `Appfile.local`

File: docs/intro/Appfile.md [hunk 2]
```diff
@@ -63,3 +63,49 @@ If you want to access those values from within your `Fastfile` use
...
+```
+
+~/.bashrc of user 2
+```bash
+export FASTLANE_USER="user2@example.org"
+```
+
+#### Using a local `Appfile` that's not checked into git
+
+If you don't want to modify the `.bashrc`, you can also use a local Appfile, that you don't commit to version control.
```
Comment (milch) [inline docs/intro/Appfile.md]: I don't think the second comma is needed here

File: docs/intro/Appfile.md (hunk 1 above)
Comment (KrauseFx) [inline docs/intro/Appfile.md]: Nice catch

File: docs/intro/Appfile.md [hunk 3]
```diff
@@ -63,3 +63,57 @@ If you want to access those values from within your `Fastfile` use
 identifier = CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)
 team_id = CredentialsManager::AppfileConfig.try_fetch_value(:team_id)
 ```
+
+### Multiple users configuration
+
+_fastlane_ supports having multiple users, depending on which machine it is currently running on.
+
+#### Using environment vairables
```
Comment (milch) [inline docs/intro/Appfile.md]: "vairables"

**Comment by KrauseFx** [comment]
Thanks 👍

**Pull request merged**